Packs/CortexAttackSurfaceManagement/Scripts/RemediationPathRuleEvaluation/RemediationPathRuleEvaluation.yml

```diff
@@ -0,0 +1,39 @@
+args:
+- description: Alert's Severity
+  name: severity
+- description: Alert's Remote IP
+  name: ip
+- description: Is this in a development environment?
+  name: development_environment
+- description: Is this Cloud Managed?
+  isArray: true
+  name: cloud_managed
+- description: Has a Service Owner been Identified?
+  isArray: true
+  name: service_owner_identified
+- description: Includes Cloud and Xpanse tags
```

Comment:
Should this be:
A comma-separated list of cloud and Xpanse tags.

Reply:
this is an array